### ISSUE TITLE
Use of dot notation 'default' breaks IE8

### ISSUE THREAD

**Issue by kevana** [rackt/react-redux#232]
Looks like this happened in the 4.0.1 release, might be related to the Babel 6 upgrade but I'm not positive.

dist/react-redux.js:
```
	function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

	exports.Provider = _Provider2.default;
	exports.connect = _connect2.default;
```

**Comment by kevana** [comment]
I think adding es3 transforms (es3-member-expression-literals, es3-property-literals) to the babel config will fix it, I'll check tonight.

**Comment by gaearon** [comment]
I'm on it

**Comment by kevana** [comment]
Thanks :+1:

**Comment by gaearon** [comment]
Unfortunately those transforms aren't currently helpful because of a bug: https://phabricator.babeljs.io/T2817. I'll have to go back to using CommonJS internally until this is fixed.

**Comment by kevana** [comment]
Ok, it's not a high priority issue for me, the webpack workaround isn't too bad:

```
// webpack config for string-replace-webpack-plugin
{
  pattern: /\.default(?=[^\w$])/g,
  replacement: function () {
    return "['default']"
  }
},
{
  pattern: /\{\sdefault:/g,
  replacement: function () {
    return "{ 'default':"
  }
}
```

**Comment by gaearon** [comment]
It's still a breaking change we need to fix though. Which is not fun. :-( I think we'll go with requiring ES5 environment in the next version, and will ask people to `es3ify` or equivalent if they need IE8.

**Comment by gaearon** [comment]
Please verify whether this is fixed in 4.0.6.

**Comment by kevana** [comment]
Verified fixed, thanks.
